src/pages/ReservationStatusPage/ReservationStatusPage.jsx

```diff
@@ -1,28 +1,144 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { useSearchParams } from "react-router-dom";
+import axios from "axios";
+import ReservationCard from "./components/ReservationCard";
 import styled from "styled-components";
-import ExpireReservationCard from "../ProfilePage/components/ExpireReservationCard";
 
-const ProfilePage = () => {
+const ReservationStatusPage = () => {
+  const [reservationList, setReservationList] = useState([]);
+  const [currentTab, setCurrentTab] = useState("전체");
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const setParams = title => {
+    if (currentTab === "전체") {
+      searchParams.delete("isExpired");
+      searchParams.delete("isMatch");
+      setSearchParams(searchParams);
+    } else if (currentTab === "일반") {
+      searchParams.set("isExpired", "0");
+      searchParams.set("isMatch", "0");
+      setSearchParams(searchParams);
+    } else if (currentTab === "매치(Host)") {
+      searchParams.set("isExpired", "0");
+      searchParams.set("isMatch", "1");
+      setSearchParams(searchParams);
+    } else if (currentTab === "매치(Guest)") {
+      searchParams.delete("isMatch");
+      searchParams.set("isExpired", "0");
+      setSearchParams(searchParams);
+    }
```

Comment:
```suggestion
    if (currentTab === "전체") {
      searchParams.delete("isExpired");
      searchParams.delete("isMatch");
    } else if (currentTab === "일반") {
      searchParams.set("isExpired", "0");
      searchParams.set("isMatch", "0");
    } else if (currentTab === "매치(Host)") {
      searchParams.set("isExpired", "0");
      searchParams.set("isMatch", "1");
    } else if (currentTab === "매치(Guest)") {
      searchParams.delete("isMatch");
      searchParams.set("isExpired", "0");
    }
    setSearchParams(searchParams);
```
중복되는 요소는 이렇게 하나로 빼서 관리해주세요